lib/compute-download-urls.js

```diff
@@ -231,6 +240,36 @@ function getWindowsFirefoxDriverArchitecture(version, wantedArchitecture) {
   return 'win.zip';
 }
 
+function getEdgeDriverDirectory(version) {
+  var release = parseInt(version)
+  switch (release) {
+    case 15063:
+      return '3/4/2/342316D7-EBE0-4F10-ABA2-AE8E0CDF36DD'
+    case 14393:
+      return '3/2/D/32D3E464-F2EF-490F-841B-05D53C848D15'
+    case 10586:
+      return 'C/0/7/C07EBF21-5305-4EC8-83B1-A6FCC8F93F45'
+    case 10240:
+      return '8/D/0/8D0D08CF-790D-4586-B726-C6469A9ED49C'
```

Comment:
Gross! This is going to be a headache to maintain if there is a lot of edge drivers released. Don't think there is really anything that can be changed here; just me wishing that Microsoft had the version number in the path instead of uuids.